src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -265,16 +268,18 @@ class Tooltip extends React.Component<Props, State> {
                 background={(popperStyle as React.CSSProperties)?.background || '#000'}
               />
             </TooltipContent>
-          )}
-        </Popper>,
-        this.getPortal(usesGlobalPortal)
-      );
-    }
+          </PositionWrapper>
+        )}
+      </Popper>
+    ) : null;
 
     return (
       <Manager>
         <Reference>{({ref}) => this.renderTrigger(children, ref)}</Reference>
-        {tip}
+        {ReactDOM.createPortal(
+          <AnimatePresence>{tip}</AnimatePresence>,
```

Comment:
do we want to render `AnimatePresence` even if the tip is null?

Reply:
Yeah you have to, otherwise there's no way to keep the component in the DOM until it's animated out